src/app/src/main/java/com/example/myapplication/Facility.java

```diff
@@ -1,5 +1,7 @@
 package com.example.myapplication;
 
+import androidx.fragment.app.Fragment;
```

Comment:
I assume this was accidental?

Reply:
Yes, thank you for catching it. I'll update it in main unless there's a way to fix rn?